### PR TITLE
feat: Add support for percentiles TSV output

### PR DESCRIPTION
**Pull request by hassila**
## Description

Add support for percentiles 0-100 with headers in TSV format.

Addresses https://github.com/ordo-one/package-benchmark/issues/90

Cleaned up naming of output formats.

## How Has This Been Tested?

Manually tested with youplot.

## Minimal checklist:

- [x] I have performed a self-review of my own code
- [x] I have added `DocC` code-level documentation for any public interfaces exported by the package
- [ ] I have added unit and/or integration tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by hassila** [comment]
Current dependencies on/for this PR:
* main
  * **PR #96** <a href="https://app.graphite.dev/github/pr/ordo-one/package-benchmark/96" target="_blank">Graphite</a> 
    * **PR #97** <a href="https://app.graphite.dev/github/pr/ordo-one/package-benchmark/97" target="_blank">Graphite</a> 
      * **PR #99** <a href="https://app.graphite.dev/github/pr/ordo-one/package-benchmark/99" target="_blank">Graphite</a> 
        * **PR #100** <a href="https://app.graphite.dev/github/pr/ordo-one/package-benchmark/100" target="_blank">Graphite</a> 
          * **PR #102** <a href="https://app.graphite.dev/github/pr/ordo-one/package-benchmark/102" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/ordo-one/package-benchmark/102?utm_source=stack-comment).

**Comment by hassila** [comment]
image

File: Plugins/Benchmark/BenchmarkPlugin.swift [hunk 1]
```diff
@@ -139,10 +139,12 @@ import PackagePlugin
...
         try filteredTargets.forEach { target in
-            if quietRunning == 0 {
-                print("Building \(target.name)")
+            if outputFormat == .text {
+                if quietRunning == 0 {
+                    print("Building \(target.name)")
+                }
             }
-
+
```
Comment (dimlio) [inline Plugins/Benchmark/BenchmarkPlugin.swift:147]: Hmm, `git` usually complains about such lines...

File: Plugins/Benchmark/Command+Helpers.swift [hunk 2]
```diff
@@ -22,17 +22,22 @@ enum Command: String, CaseIterable {
...
     case markdown
-    /// Influx data import
+    /// Influx data format
     case influx
-    case percentiles
-    case tsv
-    case jmh
-    /// The encoded representation of the underlying histograms capturing the benchmark data.
-    case encodedHistogram
+    case jmh // JMH format consumable by http://jmh.morethan.io
```
Comment (dimlio) [inline Plugins/Benchmark/Command+Helpers.swift:31]: Maybe put description on a separate line above for consistency?

**Comment by hassila** [comment]
@hassila queued this pull request to merge with [Graphite](https://app.graphite.dev/github/pr/ordo-one/package-benchmark/102).

**Comment by hassila** [comment]
[Graphite](https://app.graphite.dev/github/pr/ordo-one/package-benchmark/102) rebased this pull request as part of a merge.

**Comment by hassila** [comment]
@hassila merged this pull request with [Graphite](https://app.graphite.dev/github/pr/ordo-one/package-benchmark/102).

**Pull request merged**